stylus-proc/src/macros/public/mod.rs

```diff
@@ -237,7 +237,7 @@ fn verify_sol_name(
         Some("receive")
     } else if name_low == "fallback" && !matches!(kind, FnKind::Fallback { .. }) {
         Some("fallback")
-    } else if (name_low == "constructor" || name_low == "stylusconstructor")
+    } else if (name_low == "constructor" || name_low == "stylus_constructor")
```

Comment:
I don't understand why selector_override lowercase is used here instead of the original selector_override.

Reply:
I don't understand 😕 

`selector_override` is not converted to Case::Camel, only the `name`, right?
Is it enforced somewhere that selector_override must follow camel case? I didn't find it.
So if `stylus_constructor` is provided through the selector_override then name_low will be `stylus_constructor`.
But by your comment I am not sure if that is the intended behavior.
Am I missing something?

If those assumptions above are correct, if I understood it correctly the audit reported that this selector_override could clash with [this](https://github.com/OffchainLabs/stylus-sdk-rs/blob/6111e941e5eab9dfa5050c8fbdd25df5738fbf63/stylus-sdk/src/abi/internal.rs#L14).

Reply:
I think you are write. I thought CONSTRUCTOR_BASE_NAME was `stylusConstructor` (in camel case).

Reply:
Makes sense 🙂, thanks!